### ISSUE TITLE
Issue with metadata

### ISSUE THREAD

**Issue by notSeaSmoke** [Inpacchi/Manga-Tagger#10]
So this is what my comicinfo.xml looks like
```
<?xml version="1.0" ?>
<ComicInfo xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <!--Generated by Manga Tagger, an Endless Galaxy Studios project-->
   <Series>Temple</Series>
   <Number>001</Number>
   <Summary>Not knowing what to do with his worldly desires, Akagami Akemitsu aims to live a stoic life. However, the thick karmic blood flowing from him will not permit it! (Source: Kodansha, translated)</Summary>
   <Year>2018</Year>
   <Month>9</Month>
   <Writer>Kimitake Yoshioka</Writer>
   <Penciller>Kimitake Yoshioka</Penciller>
   <Inker>Kimitake Yoshioka</Inker>
   <Colorist>Kimitake Yoshioka</Colorist>
   <Letterer>Kimitake Yoshioka</Letterer>
   <CoverArtist>Kimitake Yoshioka</CoverArtist>
   <Publisher>Comic Days</Publisher>
   <Genre>Comedy,Ecchi,Romance,Harem,Slice of Life</Genre>
   <Web>https://myanimelist.net/manga/120643/Temple</Web>
   <LanguageISO>en</LanguageISO>
   <Manga>Yes</Manga>
   <Notes>Scraped metadata from AniList and MyAnimeList (using Jikan API) on 2020-12-12 09:18 PM IST</Notes>
</ComicInfo>
```
Everything seems fine, but it's not being read by comicrack. It looks like [this](https://files.catbox.moe/kc0eq7.png)
Tried using Komga as well, that's not reading anything either. Is there something wrong with how the metadata is being written or is something wrong on my end?

**Comment by Inpacchi** [comment]
Hey. Just wanted to let you know I am working on debugging this issue - my weekend was hectic, but back to the weekday grind so I'll have time to work on this.

**Comment by Inpacchi** [comment]
Fixed in 367bb4a4e8e4706b1316122b019141bec509a92b

Go figure, the metadata file was named as comicinfo.xml when it's supposed to be ComicInfo.xml. And go figure that Calibre reads it JUST FINE, but ComicRack and Komga won't.
